### ISSUE TITLE
default application links are openend in new window with new profile

### ISSUE THREAD

**Issue by sluTTYusr** [henrypp/chrlauncher#2]
Hi Henry++,

thanks for the Chromium launcher.
I have one issue that keeps me from being totally happy with it ;)

I cannot get Chromium to open links from Thunderbird with my default profile of the portable installation.

In 'chrlauncher.ini' I removed '--no-default-browser-check' from 'ChromiumCommandLine='.
Accordingly, I was asked if I want to set Chromium as the default browser which I OK'd.

Now, whenever I click a link in Thunderbird it opens a new Chromium window using the default Chromium profile path (C:\Users\%USERNAME%\AppData\Local\Chromium\User Data\Default)

I tried 'mklink /d C:\Users\%USERNAME%\AppData\Local\Chromium\User Data <location of the portable installation>\64\Chromium\profile' resulting in an 'cannot load your profile properly' error.

Next try was setting 
ChromiumCommandLine=--user-data-dir="C:\Users\%USERNAME%\AppData\Local\Chromium\User Data\"
in chrlauncher.ini
Result was a permission error for the profile path on the system drive.

Do you have an idea how to solve this?

Thanks!

**Comment by henrypp** [comment]
In next version be possibility to set chrlauncher as default browser. Just wait.

**Comment by henrypp** [comment]
fixed. see https://github.com/henrypp/chrlauncher/blob/master/bin/SetDefaultBrowser.bat

**Comment by sluTTYusr** [comment]
Doesn't work for me.
Ran the batch file as Administrator user. The last 5 reg add commands returned 'access denied".
If I click a link in another app that is supposed to open in the default browser the launcher starts, download progress bar runs from left to right, window closes, no browser window appears.

**Comment by henrypp** [comment]
fixed. download "bin\RegistryFix.reg" & "bin\SetDefaultBrowser.bat" and access denied will gone.

**Comment by sluTTYusr** [comment]
The access denied error is gone, however the 'fix' doesn't work.
If I klick a HTML link in Thunderbird or any other application, the launcher starts, download progress bar runs from left to right, window closes, no new browser window or tab appears.
It does not matter if a Chromium window is already open or not.

**Comment by henrypp** [comment]
fixed. i reworked association script, but now supported only Win 7 and above.

**Comment by sluTTYusr** [comment]
Hi Henry, I tried your launcher again...
Uninstalled other chromium installation, used your launcher to install stable-codecs-nosync version.
Then started SetDefaultBrowser.bat (as admin).
When I now klick a link in any other application nothing happens. The link is not opened with any browser. I tried using the RegistryCleaner and then ran SetDefaultBrowser.bat again. Still does not work.
What can I do to set it to open links from other applications?

**Comment by henrypp** [comment]
set chrlauncher as default bowser in control panel:
![ewweqq](https://cloud.githubusercontent.com/assets/3902025/14140952/e573e810-f69d-11e5-9759-1fe9b1040eac.png)

**Comment by sluTTYusr** [comment]
Great! It worked!
Thank you!

**Comment by Tom4tot** [comment]
Hi, if I may, I'd like to reopen this issue but I'm not sure you can do anything about it.

Since a few updates, on Windows 10, it's impossible for me to pick chrlauncher (which launches my right chromium profile) as my default browser.

Right now, when I go to Windows settings, I can only pick "Chromium" which opens a new chrome profile (so I don't have access to all my settings / data as well as it doesn't open my taskbar shortcut but a new instance of Chrome).

See screenshot 1.
![screenshot_1](https://user-images.githubusercontent.com/32274750/48262213-d6bfca00-e475-11e8-8b55-262fbda0f136.png)

I can manually pick chrlauncher for file type but then when I open a link on another application it still opens links in a new chrome profile.

In "default apps by protocol", it's the same as on screenshot 1: for HTTP and HTTPS, I can't manually pick a program.

See screenshot 2.
![screenshot_2](https://user-images.githubusercontent.com/32274750/48262215-d7586080-e475-11e8-862e-1de82727e32e.png)

So basically, to sum up, Windows is really limitating users right now about default programs.

1) Am I doing something wrong?
2) Or is there a way to bypass it? Maybe changing the Chromium shortcut so it would open my profile. I already had more or less similar issues but you helped to fix them: https://github.com/henrypp/chrlauncher/issues/87


Thanks in advance for your help and for all your wonderful softwares in general. :)
